docs/marionette.region.md

```diff
@@ -492,43 +506,45 @@ view.supportsDestroyLifecycle = true;
 ### Example Event Handlers
 
 ```js
-MyApp.mainRegion.on("before:show", function(view, region, options){
+var mainRegion = myView.getRegion('mainRegion');
+
+mainRegion.on("before:show", function(view, region, options){
   // manipulate the `view` or do something extra
   // with the `region`
   // you also have access to the `options` that were passed to the Region.show call
 });
 
-MyApp.mainRegion.on("show", function(view, region, options){
+mainRegion.on("show", function(view, region, options){
   // manipulate the `view` or do something extra
   // with the `region`
   // you also have access to the `options` that were passed to the Region.show call
 });
 
-MyApp.mainRegion.on("before:swapIn", function(view, region, options){
+mainRegion.on("before:swapIn", function(view, region, options){
```

Comment:
swapIn / swapOut is gone now too.   They were redundant somehow and aren't particularly useful for animation as initially intended.